doc/nrf/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_architecture_ipc.rst

```diff
@@ -104,19 +106,21 @@ Some of the channels in BELLBOARDs and VEVIFs are reserved for functions differe
 MAC, Network, Transport
 =======================
 
+.. to review
+
 The layers responsible for maintaining a stable full-duplex stream of data between two cores (MAC, Network, Transport) are implemented in an IPC transport library.
 The default IPC transport library for nRF54H20 is :ref:`ICMsg <zephyr:ipc_service_backend_icmsg>`.
 This lightweight library maintains the connection between a pair of cores for each IPC instance with minimal memory overhead.
-Each IPC instance requires an instance of the icmsg library.
-Each icmsg library instance requires the following:
+Each IPC instance requires an instance of the ICMsg library.
```

Comment:
It is actually icbmsg between cores and icmsh between app and pepper. You can link library here